pom.xml

```diff
@@ -9,9 +9,42 @@
         <version>9</version>
     </parent>
 
-    <groupId>uk.co.qualitycode</groupId>
+    <groupId>io.github.excitedkettle</groupId>
     <artifactId>functional-utils</artifactId>
-    <version>2.0-SNAPSHOT</version>
+    <version>2.0</version>
```

Comment:
It got super grumpy if I added -SNAPSHOT to the build name, something to look into.